internal/pkg/agent/application/monitoring/liveness.go

```diff
@@ -0,0 +1,85 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package monitoring
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/elastic/elastic-agent-client/v7/pkg/client"
+)
+
+const formValueKey = "failon"
+
+type LivenessFailConfig struct {
+	Degraded    bool `yaml:"degraded" config:"degraded"`
+	Failed      bool `yaml:"failed" config:"failed"`
+	Coordinator bool `yaml:"coordinator" config:"coordinator"`
```

Comment:
```suggestion
```

You don't actually need this since this check can't be turned off. If you keep it, it should be renamed to Heartbeat.